### PR TITLE
chore(map): add orgs from Portugal

### PR DESCRIPTION
**Pull request by puzzledbytheweb**
## Description
Added some organizations to Portugal, since we are lacking there.
## Motivation
We'll be rolling out to Portugal but if one sees the map it looks like we are not active there
## Release Checklist

- [ ] This code has unit tests
- [x] I have a plan to verify that these changes are working as expected after deploy
- [x] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

### CODE REVIEW

File: map/src/data/markers.ts [hunk 1]
```diff
@@ -62,6 +62,32 @@ export interface MarkerInfo {
...
+      description: 'Porto, Foz do Douro, Portugal',
+      lat: 41.1675792,
+      lng: -8.7027557,
+      serviceRadius: 2000,
+    },
+    ALL: {
+      description: 'PT',
+      lat: 54.200981,
+      lng: -4.513111,
+      serviceRadius: 280610,
```
Comment (s0) [inline map/src/data/markers.ts:88]: It looks like these are the coordinates for all of the UK not Portugal

Comment (puzzledbytheweb) [reply map/src/data/markers.ts:88]: Crap, sorry -.-

Comment (puzzledbytheweb) [reply map/src/data/markers.ts]: Fixed here - https://github.com/reach4help/reach4help/pull/369/commits/e3f0027fcc01b97173335afe6eb2e83c35650463

File: map/src/data/markers.ts [hunk 2]
```diff
@@ -62,6 +62,32 @@ export interface MarkerInfo {
...
+      description: 'Porto, Foz do Douro, Portugal',
+      lat: 41.1675792,
+      lng: -8.7027557,
+      serviceRadius: 2000,
+    },
+    ALL: {
+      description: 'PT',
+      lat: 37.1458994,
+      lng: -23.3165484,
+      serviceRadius: 280610,
```
Comment (s0) [inline map/src/data/markers.ts:88]:
```suggestion
      description: 'PT',
      lat: 39.475803,
      lng: -8.591411,
      serviceRadius: 311000,
```

Comment (s0) [reply map/src/data/markers.ts:88]: Unless you want to explicitly include the islands and not the mainland, in which case the radius needs to be a lot bigger :)

**Pull request merged**